services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/modules/db/__init__.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import FastAPI
+from servicelib.db_async_engine import close_db_connection, connect_to_db
+
+
+def setup(app: FastAPI):
+    async def on_startup() -> None:
+        await connect_to_db(app)
+
+    async def on_shutdown() -> None:
```

Comment:
At some point we need to migrate to https://fastapi.tiangolo.com/advanced/events/